samples/contrib/pytorch-samples/Pipeline-Cifar10.ipynb

```diff
@@ -209,7 +209,7 @@
     "DEPLOY_NAME=\"torchserve\"\n",
     "MODEL_NAME=\"cifar10\"\n",
     "ISVC_NAME=DEPLOY_NAME+\".\"+NAMESPACE+\".\"+\"example.com\"\n",
-    "INPUT_REQUEST=\"https://raw.githubusercontent.com/kubeflow/pipelines/master/samples/contrib/pytorch-samples/cifar10/input.json\""
+    "INPUT_REQUEST=\"https://raw.githubusercontent.com/kubeflow/pipelines/0abe0f0308d7122d801b1b19dda1fb89fb01f87d/samples/contrib/pytorch-samples/cifar10/input.json\""
```

Comment:
URL is from a commit branch, can we use the one from master branch?

Reply:
Modified the input request file in this PR. Will update with master branch URL in another PR after merge.